src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -167,6 +181,49 @@ def event_streams_table(
     return _copy_table("event_streams", query, full_path)
 
 
+@register(
+    "event_streams_by_activation_table",
+    "1.0",
+    format="csv",
+    description="Data on event_streams used by each activation",
+)
+def event_streams_by_activation_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    activations = models.Activation.objects.filter(
+        Q(created_at__gt=since, created_at__lte=until)
+        | Q(modified_at__gt=since, modified_at__lte=until)
+    ).distinct()
+
+    event_streams = models.EventStream.objects.none()
+    for activation in activations:
+        event_streams |= activation.event_streams.all()
+
+    if not bool(event_streams):
+        return
+
+    event_streams = event_streams.extra(
+        select={
+            "activation_id": "core_activation_event_streams.activation_id",
+            "event_stream_id": "core_eventstream.id",
+        }
+    ).values(
+        "activation_id",
+        "event_stream_id",
+        "name",
+        "event_stream_type",
+        "eda_credential_id",
+        "owner",
+        "events_received",
+        "last_event_received_at",
+        "organization_id",
```

Comment:
Im wondering if we actually intend to be capturing all these values for our analytics? Or if instead we should only capture a subset of these fields? 

Fields like name, owner and organization_id, I'm not sure we would want to submit as part of our analytics dat, as I'm thinking there is a chance this could be sensitive data? 

Im thinking we would need to confirm with @Ladas to be sure of the exact fields we want to collect and send?

Reply:
@zkayyali812 Yes, you're right, we need to confirm with @Ladas and @josephpisciotta . 
The reason I added some extra data is for some other MVPs stories, such as: 
```
- I wish to know the amount of events fired into eventstreams;
- I wish to know the number of event streams customers have;
- I wish to know the number of event streams per rulebook activation;
```

This single collected table may be enough for these cases.

Reply:
yeah, we shouldn't collect owner. Even if it's just user_id, it's considered PII

Reply:
@Ladas The owners' info was removed.